03-specifying-sample-designs.Rmd

```diff
@@ -687,11 +687,11 @@ des_bs <- dat %>%
 
 ```
 
-Note that the scale is usually provided in the documentation and is a constant, so it is not provided as a variable in the tibble.
+Note that the scale (\alpha) is usually provided in the documentation and is a constant, so it is not provided as a variable in the tibble.
 
 #### Example {-}
 
-Returning to the api example, bootstrap weights were constructed for a one-cluster design. Fifty replicate weights were created on a dataset `apiclus1_slim` which has some familiar variables, including cds, dnum, fpc, and pw but now additionally includes bootstrap weights pw1, ..., pw50. The scale $(\alpha)$ is $15/(14*49)=0.02186589$
+Returning to the api example, bootstrap weights were constructed for a one-cluster design. Fifty replicate weights were created on a dataset `apiclus1_slim` which has some familiar variables, including cds, dnum, fpc, and pw but now additionally includes bootstrap weights pw1, ..., pw50. The scale $(\alpha)$ is $15/(14*49)=0.02186589$ 
 
 ```{r}
 #| label: samp-des-genbs
```

Comment:
I think that this code should be included.  It's very confusing when reading it and we want people to be able to follow along.